FileUploadApi.md

```diff
@@ -0,0 +1,263 @@
+# File Upload API
+
+## Introduction
+
+This API is responsible for uploading files to storage. 
+mod-data-import module use uploaded files for parsing and importing files to storage and another FOLIO modules usage.
```

Comment:
It would be great to highlight such references to code for readability purpose. For example choose betwee two way - either surrounding it with \`str\` or with \_str>\_. Please chack other places.